com.unity.shadergraph/Editor/GraphDeltaRegistry/GraphDelta/GraphHandler.cs

```diff
@@ -31,23 +29,42 @@ public string ToSerializedFormat()
             return EditorJsonUtility.ToJson(graphDelta.m_data, true);
         }
 
-        internal NodeHandler AddNode<T>(string name, Registry.Registry registry) where T : Registry.Defs.INodeDefinitionBuilder => graphDelta.AddNode<T>(name, registry);
+        internal NodeHandler AddNode<T>(string name, Registry registry) where T : INodeDefinitionBuilder =>
+            graphDelta.AddNode<T>(name, registry);
 
-        public NodeHandler AddNode(RegistryKey key, string name, Registry.Registry registry) => graphDelta.AddNode(key, name, registry);
-        public NodeHandler AddContextNode(RegistryKey key, Registry.Registry registry) => graphDelta.AddContextNode(key, registry);
-        public bool ReconcretizeNode(string name, Registry.Registry registry) => graphDelta.ReconcretizeNode(name, registry);
+        public NodeHandler AddNode(RegistryKey key, string name, Registry registry) =>
```

Comment:
Style wise, for multi-line inline functions; I'm wondering if it's more clear if the => appears on the following line.

Reply:
Stylistically, I have been updating inline functions to be this way because I feel it makes them read more like functions or methods. Also, one line in-lines can be pretty long.